static/js/components/kto-link.scss

```diff
@@ -17,7 +17,7 @@
     content: '';
     width: 100%;
     height: 1px;
-    background: #fff;
+    background: #ffffff;
```

Comment:
what's happened? O_o

Reply:
nothing special, I just reverted the file to the previous state.